src/sentry/auth/access.py

```diff
@@ -119,11 +129,17 @@ def has_scope(self, scope: str) -> bool:
         """
         return scope in self.scopes
 
+    # TODO(cathy): remove this
     def get_organization_role(self) -> OrganizationRole | None:
         if self.role is not None:
             return cast(OrganizationRole, organization_roles.get(self.role))
         return None
 
+    def get_organization_roles(self) -> Iterable[OrganizationRole]:
```

Comment:
Maybe we could make this a property too?
```
@property 
def organization_roles(self):
...
```

Reply:
i'm going to come back to this when i remove the other properties, so i'll think about replacing it then